src/views/account/UserProfile.vue

```diff
@@ -17,11 +18,64 @@ const userProfile = ref<UserProfile>({
   institution: "",
   last_name: "",
   location: "",
-  password: "",
   profile_image: "",
   timezone: "",
 });
 
+const passwordForm = ref({
+  confirm_password: "",
+  new_password: "",
+  old_password: "",
+});
+
+const passwordRules: FormRules = {
+  confirm_password: {
+    required: true,
+    trigger: ["blur", "input"],
+    validator: (rule, value) => {
+      if (value !== passwordForm.value.new_password) {
+        return new Error("Passwords do not match");
+      }
+      return true;
+    },
+  },
+  new_password: {
+    required: true,
+    trigger: ["blur", "input"],
+    validator: (rule, value) => {
+      if (!value) {
+        return new Error("Please enter a new password");
+      }
+      // Validate value has 8 characters
+      if (value.length < 8) {
+        return new Error("Password must be at least 8 characters");
+      }
+      // Validate value has at least 1 uppercase character
+      if (!/[A-Z]/.test(value)) {
+        return new Error("Password must contain at least 1 uppercase character");
+      }
+      // Validate value has at least 1 lowercase character
+      if (!/[a-z]/.test(value)) {
+        return new Error("Password must contain at least 1 lowercase character");
+      }
+      // Validate value has at least 1 number
+      if (!/[0-9]/.test(value)) {
+        return new Error("Password must contain at least 1 number");
+      }
+      // Validate value has at least 1 special character
+      if (!/[!@#$%^&*()_+\-=[\]{};':"\\|,.<>/?]/.test(value)) {
+        return new Error("Password must contain at least 1 special character");
+      }
+      return true;
```

Comment:
Add a validator rule to ensure that the new password is not the same as the old password